### ISSUE TITLE
[Feature request] Set default playback speed

### ISSUE THREAD

**Issue by qwerty12** [jarnedemeulemeester/findroid#317]
Hi,

Thank you for Findroid. As I usually watch stuff at 1.25x speed, the ability to set the default playback speed from the settings would be great.

**Comment by lu4p** [comment]
I think the player should just remember (including after app restarts) the last speed that was set instead of a default which unnecessarily complicates things.

**Comment by Patriot2407** [comment]
I have to set the speed on every single video I watch manually

**Comment by qwerty12** [comment]
If you have Findroid set to use mpv to playback video, you can put something like [`speed=1.25`](https://mpv.io/manual/stable/#options-speed) in [mpv.conf](https://github.com/jarnedemeulemeester/findroid/issues/157#issuecomment-1233894385)

**Comment by Patriot2407** [comment]
Does that keep track of my position in the video?

**Comment by qwerty12** [comment]
Yes - in this case I refer to Findroid's integrated libmpv (settings -> player), not mpv as an external player

**Comment by Patriot2407** [comment]
I added speed=1.5 and set the app to use MPV, force quit the app and reopened the app, and it was still playing at 1x

**Comment by Patriot2407** [comment]
--speed worked. Just note that the gui still shows it as 1x, but mpv is actually playing it in whatever you set.
